viewing_party/party.py

```diff
@@ -1,23 +1,176 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_info = [title, genre, rating]
+    if None in movie_info:
+        movie = None
+    else:
+        movie = {"title": title, "genre": genre, "rating": rating}
+    return movie

```

Comment:
Let's move the guard clause up to the top because we create any variables.

```suggestion
    if not title or not genre or not rating:
        return None
    movie = {"title": title, "genre": genre, "rating": rating}
    return movie
```

We want this to use a `return` statement. That way it immediately leaves the function instead of continuing